package.json

```diff
@@ -23,6 +23,7 @@
     "react-hot-toast": "^2.4.1",
     "react-icons": "^4.11.0",
     "react-router-dom": "^6.10.0",
+    "realmate": "file:",
```

Comment:
why did you added this line? what it does?

Reply:
i didnt added that, i just did npm i, i m removing this

Reply:
i have removed that line, Please check now